lighthouse-core/audits/audit.js

```diff
@@ -33,6 +35,27 @@ class Audit {
   }
 
   /**
+   * Computes and clamps the score between 0 and 100 based on the measured value, and the two control
+   * points on the log-normal distribution (point of diminishing returns and the median value).
+   *
+   * @param {number} measuredValue
+   * @param {number} diminishingReturnsValue
+   * @param {number} medianValue
+   * @return {number}
+   */
+  static computeScore(measuredValue, diminishingReturnsValue, medianValue) {
```

Comment:
that's probably good if it stays in audit.js. It would be good to note in the jsdoc description something about that it returns the percentage of sites with values higher than `measuredValue` from the distribution with the given attributes. But englished good.